episodes/04-data-structures-part2.Rmd

```diff
@@ -86,10 +86,12 @@ always do is check out what the data looks like with `str`:
 str(gapminder)
 ```
 
-We can also examine individual columns of the data frame with our `class` function:
+We can also examine individual columns of the data frame with the `class` or
+'typeof' functions.:
```

Comment:
```suggestion
'typeof' functions:
```

I'd suggest just using `class()` here. In my experience, `typeof()` is a much less common function which has some non-intuitive behaviors, for instance returning `double` instead of `numeric` and returning `integer` for factors:

```r
> typeof(factor("a"))
[1] "integer"
```

This is because `typeof()` indicates the internal type (effectively, C type) of an object, rather than its R-level class. Since it's the R-level class that determines how everything in R will treat a variable, I think it makes more sense to emphasize `class()` alone here, to avoid confusing learners with multiple representations of the same concept.

Reply:
Would you like to open a new issue about your suggestion @mikemahoney218? I think it would be better that way.